sonar-project.properties

```diff
@@ -0,0 +1,15 @@
+sonar.projectKey=alphagov_govuk-mobile-ios-onboarding
+sonar.projectName=govuk-mobile-ios-onboarding
+sonar.pullrequest.github.repository=alphagov/govuk-mobile-ios-onboarding
+sonar.projectVersion=0.0.1
+
+sonar.organization=alphagov
+
+sonar.sources=Onboarding/Sources, Example/Production
```

Comment:
How much do we care about the Example code being sonar qubed? There is no harm in it I guess as long as it doesn't become a pain or artificially reduce our unit test coverage too much to the point of spending most our time writing tests for "fake" code.

Reply:
Yeah, good point, I suppose it depends on the scope of the tests for the example app. If we consider some of them part of the test suite for the actual production code then we should include, if they're merely for ensuring the example app runs then we shouldn't include. If your opinion is the latter then happy to remove.

Reply:
Lets leave in for now, easier to undo it than add it in if we change our minds